packages/kbn-management/settings/application/services.tsx

```diff
@@ -6,17 +6,37 @@
  * Side Public License, v 1.
  */
 
-import React, { FC } from 'react';
+import React, { FC, useContext } from 'react';
 
 import {
   FormProvider,
   FormKibanaProvider,
   type FormKibanaDependencies,
   type FormServices,
 } from '@kbn/management-settings-components-form';
+import { UiSettingMetadata } from '@kbn/management-settings-types';
+import { IUiSettingsClient } from '@kbn/core-ui-settings-browser';
+import { normalizeSettings } from '@kbn/management-settings-utilities';
+import { Subscription } from 'rxjs';
 
-export type SettingsApplicationServices = FormServices;
-export type SettingsApplicationKibanaDependencies = FormKibanaDependencies;
+export interface Services {
+  getAllowlistedSettings: () => Record<string, UiSettingMetadata>;
```

Comment:
Here, we can keep the parameters and return types local to the `@kbn/management-settings-*` packages.